### PR TITLE
Creating the start of a the new CONTRIBUTING.md as mentioned in #116

### PR DESCRIPTION
**Pull request by DR-DinoMight**
Creating the start of a the new CONTRIBUTING.md as mentioned in #116 by @isabellabrookes I added a little intro to hopefully ease any new comers nerves into contributing to open source project, please remove/alter if we fell its not right for the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@
   <a href="#-how-to-contribute">How to Contribute</a>
   <span> · </span>
   <a href="#current-contributors">Current Contributors</a>
```
Comment (isabellabrookes) [inline README.md:25]: These will need to be updated to link to the new file. 👍

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Contributing
+
+## The Mantra
+
+We are great believers in open source and the great community that it builds, we feel that is anyone's right to participate in the development of Grüvee, this means everyone should have equal oppertunity to contrubute to the awesome world of open source, and at the same time be treated with the highest levels of respect.
+
+At the end of the day we're all here to build something awesome, so let's go ahead and make something awesome!
+
```
Comment (isabellabrookes) [inline CONTRIBUTING.md:8]: This is cute as heck. 😻 🖤

File: README.md (hunk 1 above)
Comment (DR-DinoMight) [reply README.md]: Done and just updated the PR with the new links

File: CONTRIBUTING.md (hunk 2 above)
Comment (DR-DinoMight) [reply CONTRIBUTING.md:8]: Glad it's okay! I really wanted something to just lay down the expectations of contributing  but didn't want it sound too formal.

**Comment by isabellabrookes** [comment]
@all-contributors please add @DR-DinoMight  for docs

**Comment by isabellabrookes** [comment]
@all-contributors please add @DR-DinoMight for docs

**Pull request merged**